Sources/Basics/FileSystem/FileSystem+Extensions.swift

```diff
@@ -242,6 +242,12 @@ extension FileSystem {
             try self.dotSwiftPM.appending("cache")
         }
     }
+    
+    private var dotSwiftPMInstalledBinsDir: AbsolutePath {
+        get throws {
+            try self.dotSwiftPM.appending("bin")
+        }
+    }
```

Comment:
Let's move this property to the extension below you've just added on the line 279 so that code related to this directory is grouped.